datasets/ONS-UK-trade-in-services-by-business-characteristics/spec.md

```diff
@@ -0,0 +1,50 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1671\cocoasubrtf600
+{\fonttbl\f0\fmodern\fcharset0 Courier;}
+{\colortbl;\red255\green255\blue255;\red0\green0\blue0;\red27\green31\blue34;\red255\green255\blue255;
+}
+{\*\expandedcolortbl;;\cssrgb\c0\c0\c0;\cssrgb\c14118\c16078\c18039;\cssrgb\c100000\c100000\c100000;
+}
+\paperw11900\paperh16840\margl1440\margr1440\vieww21020\viewh12980\viewkind0
+\deftab720
+\pard\pardeftab720\partightenfactor0
+
+\f0\fs26 \cf0 \expnd0\expndtw0\kerning0
+<!-- #region -->\
+# COGS Dataset Specification\
+----------\
+\
+## \cf3 \cb4 ONS UK trade in services by business characteristics\cf0 \cb1 \
+[Landing Page](https://www.ons.gov.uk/businessindustryandtrade/internationaltrade/datasets/uktradeingoodsbybusinesscharacteristics)\
+\
+----------\
+\
+### Stage 1. Transform\
+\
+#### Sheet: Sheet1 \
+    \
+Transform notes   \
+\
+#### Table Structure\
+\
+		Columns go here\
+\
+#### DE Stage one notes \
+Notes go here.\
+\
+### Stage 2 - Harmonisation\
+\
+Exports (\'a3m) and Imports (\'a3m) should be combined into a dimension and the values should go in the \'91Value\'92 column.\
```

Comment:
In order to represent this you'll need to add the `flow` dimension - the codelist is already defined for you in https://github.com/GSS-Cogs/ref_common/blob/pmd4/codelists/flow.csv

Reply:
Thanks - originally this was a 'Direction' dimension but I've changed that to 'Flow' in the spec.